### DISCUSSION TITLE
CVE-2022-23529 on jsonwebtoken <= 8.5.1

### DISCUSSION THREAD

**Discussion by odedonato** [requarks/wiki#6033] [Error / Bug Report] [answered]
By exploiting this [vulnerability](https://github.com/advisories/GHSA-27h2-hvpr-p74q), attackers could achieve remote code execution (RCE) on a server verifying a maliciously crafted JSON web token (JWT) request.

https://nvd.nist.gov/vuln/detail/CVE-2022-23529
https://thehackernews.com/2023/01/critical-security-flaw-found-in.html

**Comment by NGPixel** [comment, marked as answer]
Not relevant to this project.
